src/cgame/cg_rocket_draw.cpp

```diff
@@ -2130,17 +2130,20 @@ class PredictedMineEfficiencyElement : public HudElement
 			{
 				if        ( deltaBudget < 0 ) {
 					color = Color::Red;
-					msg = va( "<span class='material-icon error'>&#xE000;</span> You are losing build points!"
+					// Icon U+E000
```

Comment:
I think I've seen in other commits things to convert to/from UTF-8, maybe it would be possible to use them instead of manually made UTF-8 codes? (I might be wrong though)

Reply:
Ok then, but I would not expect anyone to be able to guess that. Is there something to do to help on that? (adding comment everywhere might be counter-productive, if this happens in other places, but I do not think I've seen other ones?)